### PR TITLE
Update FreeAgent entry

### PR DESCRIPTION
**Pull request by danaeshell**
FreeAgent now supports 2FA!  Added the details and a link to documentation.

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -26,7 +26,9 @@ websites:
       url: https://www.freeagent.com
       img: freeagent.png
       twitter: freeagent
```
Comment (Carlgo11) [inline _data/finance.yml]: Hello @danaeshell, 😄 
The Twitter tag should be removed when the `tfa` tag is changed to `Yes`. Please remove it and the build should pass 😉

File: _data/finance.yml [hunk 2]
```diff
@@ -26,7 +26,9 @@ websites:
...
-      tfa: No
+      tfa: Yes
+      software: Yes
+      docs: http://www.freeagent.com/support/kb/getting-started/two-step-verification
```
Comment (Jawshy) [inline _data/finance.yml]: This webpage also supports HTTPS which TwoFactorAuth prefers.

**Comment by stephengroat** [comment]
@danaeshell could you make the changes recommended? Otherwise, I'll backlog this PR

**Comment by danaeshell** [comment]
@stephengroat will do, thanks for the reminder!

**Comment by stephengroat** [comment]
Now you're really not going to like me ... could you also switch the doc link the `https`? thanks for the quick response!

File: _data/finance.yml [hunk 3]
```diff
@@ -25,8 +25,9 @@ websites:
     - name: FreeAgent
       url: https://www.freeagent.com
       img: freeagent.png
-      twitter: freeagent
-      tfa: No
+      tfa: Yes
+      software: Yes
+      docs: https://www.freeagent.com/support/kb/getting-started/two-step-verification
```
Comment (danaeshell) [inline _data/finance.yml:30]: @stephengroat in reply to your last PR comment - do you mean change this line away from https?  Thanks!

Comment (stephengroat) [reply _data/finance.yml:30]: I missed the change, just saw the git commit headline of `Remove twitter link`. Sorry, everything's perfect now!

Comment (danaeshell) [reply _data/finance.yml:30]: @stephengroat ah perfect, i thought that might be it!  Thank you!

**Comment by danaeshell** [comment]
Hey!  Just to check - does the 'updated site' label mean that the change has been deployed?  I just noticed that FreeAgent is still listed as not supporting 2FA. Thank you!

**Comment by stephengroat** [comment]
@danaeshell to deploy we have to merge `master` into the `gh-pages` branch. it takes a little more time and a few more people make sure all of the commits being merged in are good. no problem and we'll get on it soon!

**Comment by danaeshell** [comment]
@stephengroat oh perfect, thanks for clarifying!

**Pull request merged**